app/components/avatar/avatar.stories.tsx

```diff
@@ -23,11 +23,11 @@ export const Basic = () => {
 };
 
 export const Project = () => {
-  return <ProjectAvatar project={{ name: "Bitcoin", slug: "bitcoin" }} />;
+  return <ProjectAvatar project={{ name: "Ethereum", slug: "ethereum" }} />;
 };
 
 export const Token = () => {
-  return <TokenAvatar token={{ name: "Bitcoin", symbol: "BTC" }} />;
+  return <TokenAvatar token={{ name: "Ethereum", networkId: "1", symbol: "ETH" }} />;
```

Comment:
I'd update the typings on the TokenAvatar and ProjectAvatar props to be  like `Pick<Token, "name" | "symbol">`.

So we only type the things we know the component needs, which in this case doesn't include `networkId`.